src/components/how-to-start/HowToStart.css

```diff
@@ -0,0 +1,116 @@
+.how {
+  padding: 100px 80px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start;
+  max-width: 1440px;
+  box-sizing: border-box;
+  margin: 0 auto;
+}
+
+.how__title {
+  font-size: 48px;
+  line-height: 56px;
+  text-align: center;
+  font-weight: 400;
+  margin: 0;
+  margin-bottom: 16px;
+}
+
+.how__content {
+  display: grid;
+  grid-template-areas:
```

Comment:
Nice how you did this with grid template areas! 🎉

Reply:
Thanks!!!